app/backend/src/couchers/servicers/auth.py

```diff
@@ -154,11 +154,20 @@ def SignupFlow(self, request, context):
                     existing_user = session.execute(
                         select(User).where(User.email == request.basic.email)
                     ).scalar_one_or_none()
+                   
+
                     if existing_user:
                         context.abort(grpc.StatusCode.FAILED_PRECONDITION, errors.SIGNUP_FLOW_EMAIL_TAKEN)
                     existing_flow = session.execute(
                         select(SignupFlow).where(SignupFlow.email == request.basic.email)
                     ).scalar_one_or_none()
+                    
+                    existing_user_name = session.execute(
```

Comment:
Should be `existing_username` with `username` as one word for consistency!

Reply:
done